### ISSUE TITLE
Problem with loss

### ISSUE THREAD

**Issue by ruimaximo** [equinor/gordo#430]
![image](https://user-images.githubusercontent.com/32925235/63690529-34f34900-c80d-11e9-927b-cb2aa30fa2d9.png)

Loss of "a" and "b" should be in the same range. 

Info:
- gordo-components version: 0.28.1.dev6+gc7fcb2b.d20190826
- issue observed also for the FF.

**Comment by epa095** [comment]
Hi, in your model `y1` and `y2` are not scaled, so it makes sense that the errors are very different. The solution is to wrap the model in ` sklearn.compose.TransformedTargetRegressor`:

```python
from gordo_components import serializer
import yaml
import numpy as np
from sklearn.metrics import r2_score

config = yaml.load(
    """
    sklearn.compose.TransformedTargetRegressor:
        transformer: sklearn.preprocessing.data.MinMaxScaler
        regressor:
          sklearn.pipeline.Pipeline:
              steps:
              - sklearn.preprocessing.data.MinMaxScaler
              - gordo_components.model.models.KerasLSTMAutoEncoder:
                  kind: lstm_hourglass
                  lookback_window: 10
                  epochs: 1
    """
)
model = serializer.pipeline_from_definition(config)

X1 = numpy.random.rand(100,10)
y1 = X1

a = model.fit(X1,y1)


X2 = numpy.random.rand(100,10)*100
y2 = X2

b = model.fit(X2,y2)
```
It is my understanding that @Natalie-Caruana will write up some more examples, also including how to get the errors for the scaled y instead of the unscaled y as this provides.
